src/worker.py

```
@@ -1329,6 +1314,360 @@ async def serve_static(path: str, env):
+            "is_moving":      False,
+            "seat_id":        initial_seat_id,
+        }
+
+        try:
+            server.send(json.dumps({
+                "type":           "user_info",
+                "session_id":     session_id,
+                "participant_id": participant_id,
+                "display_name":   display_name,
+            }))
+        except Exception as exc:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)